affinity/cpp-20/dXXX1r0.md

```diff
@@ -195,58 +266,147 @@ Below *(Listing 2)* is an example of executing a parallel task over 8 threads us
 ```
 *Listing 2: Example of using the bulk_execution_affinity property*
 
+### Proposed Wording
 
-## Header `<execution>` synopsis
+The `bulk_execution_affinity_t` property is a behavioral property as defined in P0443 [[22]][p0443] which describes the guarantees an executor provides to the binding of *execution agents* created by a call to `bulk_execute` to the underlying *threads of execution* and to the locality of those *threads of execution*.
+
+The `bulk_execution_affinity_t` property provides nested property types and objects as described below, where:
+* `e` denotes an executor object of type `E`,
+* `f` denotes a function object of type `F&&`,
+* `s` denotes a shape object of type `execution::executor_shape<E>`,
+* `sf` denotes a function object of type `SF`.
+
+| Nested Property Type | Nested Property Name | Requirements |
+|----------------------|----------------------|--------------|
+| bulk_execution_affinity_t::none_t | bulk_execution_affinity_t::none | A call to `e.bulk_execute(f, s, sf)` may or may not bind the created *execution agents* to the underlying *execution resources*. The affinity binding pattern may or may not be consistent across invocations of the executor's bulk execution function.  |
```

Comment:
Requirements for `none` should just be "none."  I get what you're trying to say, but it's a lot of words to say "no requirements."  Also, does "none" mean "I am not specifying the requirements," or does it mean "I don't want affinity at all"?  Those are two different things.

Reply:
I've reduced the wording to simply say that there are no requirements on binding.

The intention was that `none` means "I am not specifying any requirements", but you make a good point that this is not the same as no affinity at all. Do you think we need to introduce an additional option? If so perhaps what is currently `none` should be something like `automatic`.